### DISCUSSION TITLE
How to change TextInput background?

### DISCUSSION THREAD

**Discussion by hansyulian** [mantinedev/mantine#5843] [Q&A] [answered]
Hi, i'm wondering is there a way for me to achieve this?
The background of the text input is changed

![image](https://github.com/mantinedev/mantine/assets/10959546/f423d55e-593d-44d0-aeca-0766bf278ad3)

Thank you

**Comment by rtivital** [comment]
`<TextInput radius="xl" placeholder="Phone Number" />`

**Reply by hansyulian** [reply]
hey sorry it maybe not that visible... actually the background of the input is `#f5f5f5`, let me paintdrop a color to make it easier:
![image](https://github.com/mantinedev/mantine/assets/10959546/2e029bf0-7e29-49f7-881c-de336e03c02e)
the red part is the one that i want to change

**Reply by rtivital** [reply, marked as answer]
https://mantine.dev/styles/styles-api/
